src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -447,12 +480,27 @@ public static String convertToStandardDuration(Long timeInMilliseconds) {
     }
 
     /**
-     * Parses a {@code LocalDateTime} object from a date time string according to a pattern.
-     * Returns {@code null} on error.
+     * Parses an {@code Instant} object from a datetime string in the {@link SystemParams#DEFAULT_DATE_TIME_FORMAT}.
      *
-     * @param dateTimeString    the string containing the date and time
-     * @param pattern           the pattern of the date and time string
-     * @return                  the parsed {@code LocalDateTime} object, or {@code null} if there are errors
+     * @param dateTimeString should be in the format {@link SystemParams#DEFAULT_DATE_TIME_FORMAT}
+     * @return the parsed {@code Instant} object, or {@code null} if there are errors
```

Comment:
An error will be thrown before this happens, right?

Reply:
I think assumption failures are used to indicate programmer error? In this case `parseInstant` should never be called on a string that might be in an incorrect format - validation should happen before it is called.

Reply:
Okay, I updated the javadoc to show `AssertionError` and removed the `return null`